docker-compose-gpu-runner-cuda-12-0.yml

```diff
@@ -69,3 +69,7 @@ services:
     environment:
       # We bind only to 127.0.0.1, so setting insecure password is acceptable here
       - POSTGRES_PASSWORD=notsecurepassword
+
+volumes:
+  postgres-data:
+  reth-data:
```

Comment:
Do we even use `docker-compose-cpu-runner.yml` and `docker-compose-gpu-runner.yml` anymore? Pretty sure those can just be deleted, but happy to update if anyone insists.